src/index.ts

```diff
@@ -77,13 +81,19 @@ export default class Combobox {
   }
 
   indicateDefaultOption(): void {
-    if (this.defaultFirstOption) {
+    if (this.firstOptionSelectionMode === 'selected') {
       Array.from(this.list.querySelectorAll<HTMLElement>('[role="option"]:not([aria-disabled="true"])'))
         .filter(visible)[0]
         ?.setAttribute('data-combobox-option-default', 'true')
     }
```

Comment:
Would it work to add an `else if (this.firstOptionSelectionMode === 'focused') {this.navigate(1)}` case here instead of defining a separate `focusDefaultOptionIfNeeded` method? That way we only have to call one function in `start` and we can't forget to add the second case if we call `indicateDefaultOption` elsewhere.

Reply:
I opt'ed for a new method because this method was _also_ being called here: https://github.com/github/combobox-nav/blob/c28985b6c7890cd53a1a04ef6892cbb39ce62dd5/src/index.ts#L136 - where we _don't_ want to call navigate on. Wdyt?

Reply:
ooh I see. I think I still would rather combine the methods, but then maybe here we add a conditional `if (this.firstOptionSelectionMode === 'active')` (or `if (this.firstOptionSelectionMode !== 'selected')`?)

Reply:
Agreed! Addressed here: https://github.com/github/combobox-nav/pull/81/commits/1ef750e2aab5c4603aca7a0a3bb9e8f0dd9c0900